scripts/jenkins/groovy/buildConfig.groovy

```diff
@@ -20,6 +20,9 @@ class BuildConfig {
 
   private static final String HADOOP_IMAGE_NAME_PREFIX = 'h2o-3-hadoop'
   private static final String HADOOP_IMAGE_VERSION_TAG = '79'
+  
+  private static final String K8S_TEST_IMAGE_VERSION_TAG = '1'
```

Comment:
I think just one version is enough

Reply:
I also vote for that simplification. Let's do it this way.